components/user/UserMilestone.js

```diff
@@ -13,6 +13,7 @@ export default function UserMilestone({ milestone, isGoal }) {
         borderColor: milestone.color,
       }}
     >
+      <div className="mt-2">
```

Comment:
If it is a change to each item, could it be added to the `li`?

Reply:
> If it is a change to each item, could it be added to the `li`?

Yes it can be added to each item in the list. Like what I mean to say is it will provide some margin-top to each item in the milestone list.